startproject/lessons/OOCSS.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>OOCSS</title>
+</head>
+<style>
+    .globalwidth {
+        width: 100%;
+    }
+    .header {
+       
+    }
+    footer {
```

Comment:
shouldn't this be a class? see line 22 in this file

Reply:
Listo, ya quedaron los cambios.